packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -82,7 +80,26 @@ export default class SelectOpener extends React.Component<SelectOpenerProps> {
         this.props.onOpenChanged(!open);
     };
 
-    renderClickableBehavior(router: any): React.ReactNode {
+    handleKeyDown: (e: React.KeyboardEvent) => void = (e) => {
+        const keyCode = e.key;
+        // Prevent default behavior for Enter key. Without this, the select
+        // is only open while the Enter key is pressed.
+        // Prevent default behavior for Space key. Without this, Safari stays in
+        // active state visually
+        if (keyCode === "Enter" || keyCode === " ") {
```

Comment:
**praise:** Thanks for adding comments per browser here! it's super helpful for understanding why we have to do this.